content/20.pharmaceuticals.md

```diff
@@ -685,7 +688,7 @@ Clinical trials that are in progress are likely to provide additional insight in
 Monoclonal antibodies have revolutionized the way we treat human diseases.
 They have become some of the best-selling drugs in the pharmaceutical market in recent years [@doi:10.1186/s12929-019-0592-z].
 There are currently 79 FDA approved mAbs on the market, including antibodies for viral infections (e.g. Ibalizumab for HIV and Palivizumab for RSV) [@doi:10.1186/s12929-019-0592-z; @doi:10.1146/annurev-immunol-032712-095916].
-Virus-specific nAbs commonly target viral surface glycoproteins or host structures, thereby inhibiting viral entry through receptor binding interference [@doi:10.3389/fmicb.2017.02323; @doi:10.1080/21645515.2017.1337614].
+Virus-specific nAbs commonly target viral surface glycoproteins or host structures, thereby inhibiting viral entry through receptor binding interference [@doi:10.3389/fmicb.2017.02323; @doi:10.1080/21645515.2017.1337614]. <!--Are these all monoclonal? If not, the flow is a bit off-->
```

Comment:
@nilswellhausen do you know the answer to this?

Reply:
As I said this section is outdated by now. I will try to get to this ASAP. Recombinantly expressed neutralizing antibodies such as REGN-COV2 (what former Pres. Trump got) are monoclonal. I think I need to do a better job saying that I focus on therapeutic antibodies and not the neutralizing antibodies that a person developes during an ongoing infection

Reply:
Thanks @nilswellhausen! I truly know nothing about this topic, I think I learned a little background for when I was reading through your text last year, but that's it!